src/olympia/addons/indexers.py

```diff
@@ -23,6 +23,8 @@ def get_mapping(cls):
             'properties': {
                 'max': {'type': 'long'},
                 'min': {'type': 'long'},
+                'max_human': {'type': 'string', 'index': 'no'},
+                'min_human': {'type': 'string', 'index': 'no'},
```

Comment:
It's only meant to be displayed back to the client in the API in the `compatibility` field, so it does not care what the version string is, it'll just store it and return it as-is.